Pidaq/Lib/dts_manager/message_config.json

```diff
@@ -0,0 +1,99 @@
+{
+    "dts": {
+        "readings": [
+            {
+                "reading": "temperatures1",
+                "message_id": 160,
+                "conversion_factor": 10
```

Comment:
Am I correct in saying that for the inverter telemetry the conversion factor stored in the "Sensor Reading" object isn't actually used?

Reply:
Yup, gonna change this so we are actually using them in the manager, and any that we don't need can be set to 1